File: MODUL-5/C/PRAK504-2310817220002-RAUDATULSHOLEHAH.c
#include <stdio.h>
int reverse(int x)
{
    // Lengkapi Function ini
    int reversed = 0;
    while (x > 0)
    {
        int jumlah = x % 10;
        reversed = (reversed * 10) + jumlah;
        x /= 10;
    }
    return reversed;
}
int main()
{
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C));
}

File: MODUL-5/PYTHON/PRAK504-2310817220002-RAUDATULSHOLEHAH.py
def reverse(x):
    reversed_jumlah = 0
    while x > 0:
        jumlah = x % 10
        reversed_jumlah = (reversed_jumlah * 10) + jumlah
        x //= 10
    return reversed_jumlah

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))